packages/squiggle-lang/src/value/index.ts

```diff
@@ -416,3 +514,60 @@ export type Value =
   | VScale
   | VDomain
   | VVoid;
+
+export const valueTypeName = (v: Value) => {
+  const names = {
+    Array: "Array",
+    Bool: "Boolean",
+    Date: "Date",
+    Dist: "Distribution",
```

Comment:
TODO: It's not clear how to differentiate Distribution types. Maybe something like, ``Dist.typeOf()`` that just is for Dists